selectel/resource_selectel_mks_cluster_v1.go

```diff
@@ -0,0 +1,527 @@
+package selectel
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/selectel/go-selvpcclient/selvpcclient/resell/v2/tokens"
+	v1 "github.com/selectel/mks-go/pkg/v1"
+	"github.com/selectel/mks-go/pkg/v1/cluster"
+	"github.com/selectel/mks-go/pkg/v1/nodegroup"
+)
+
+func resourceMKSClusterV1() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceMKSClusterV1Create,
+		Read:     resourceMKSClusterV1Read,
+		Update:   resourceMKSClusterV1Update,
+		Delete:   resourceMKSClusterV1Delete,
+		Importer: nil,
+		CustomizeDiff: customdiff.All(
+			customdiff.ComputedIf(
+				"maintenance_window_end",
+				func(d *schema.ResourceDiff, meta interface{}) bool {
+					return d.HasChange("maintenance_window_start")
+				}),
+		),
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"kube_version": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false,
+				StateFunc: func(v interface{}) string {
+					return strings.Trim(v.(string), "v")
```

Comment:
Wouldn't it be better to use `TrimPrefix` here since we're only removing one symbol only from the start of the string?
https://pkg.go.dev/strings?tab=doc#TrimPrefix
https://pkg.go.dev/strings?tab=doc#Trim